src/PhpParser/Printer/BetterStandardPrinter.php

```diff
@@ -547,8 +533,8 @@ private function cleanEndWithPHPOpenTag(string $content): string
 
     private function cleanSurplusTag(string $content): string
     {
-        $content = str_replace('<?php <?php', '<?php', $content);
-        $content = str_replace('?>?>', '?>', $content);
+        $content = str_replace('<?php', '<?php', $content);
+        $content = str_replace('?>', '?>', $content);
```

Comment:
search and replace with equal content doesn't change anything, ref https://3v4l.org/6SCit, also, clean surplus and end tag will only valid on detected re-print InlineHTML, that's why `disableIsRequireReprintInlineHTML` flag was needed, without that, that will cause random invalid change.

Reply:
This is some bug in Rector replacing itself while printing :) will fix